frontend/src/css/Generic.styl

```diff
@@ -42,6 +42,22 @@ div.mce-fullscreen
   z-index 1050
   &.mce-container
     top 61px
+/*Editing container*/
+.mce-tinymce, .mce-container-body
+  min-height 100%
+/*Editing area*/
+.mce-container-body .mce-edit-area
+  position absolute
```

Comment:
I'm not a big fan of absolute solution.
The stackoverflow answer you linked in the issue shows another solution using flexbox.
The flexbox solution is cleaner and more recent than the accepted one.
If you didn't try it, do it and if it works, prefer this one

Reply:
I already try it and re-try it but it didn't work well and I didn't find a way to fix it.

Reply:
ok then